applications/CompressiblePotentialFlowApplication/python_scripts/forces_elements_to_nodes_process.py

```diff
@@ -0,0 +1,49 @@
+import KratosMultiphysics
+from KratosMultiphysics.CompressiblePotentialFlowApplication.compute_lift_process import ComputeLiftProcess
+
+def Factory(settings, Model):
+    if(not isinstance(settings, KratosMultiphysics.Parameters)):
+        raise Exception(
+            "expected input shall be a Parameters object, encapsulating a json string")
+    return ForcesElementsToNodesProcess(Model, settings["Parameters"])
+
+# all the processes python processes should be derived from "python_process"
+
+class ForcesElementsToNodesProcess(ComputeLiftProcess):
```

Comment:
You are looping the conditions. This name is misleading. I'd suggest to change it to something like `ConditionForceToNodesProcess`. By the way, why are you deriving from the `ComputeLiftProcess`? These two utilities seem to be quite different?